### PR TITLE
boards: Pine64 PineTime DevKit0

### PR DESCRIPTION
**Pull request by rafacouto**
Board definition for PineTime DevKit0 by Pine64 vendor. It is based on information published in https://wiki.pine64.org/index.php/PineTime

Signed-off-by: Rafa Couto <caligari@treboada.net>

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]
Docs are not quite ready for review, but here's a fix for the doc build failure.

**Comment by DWagenk** [comment]
received one of the devkits today and will collaborate on getting zephyr running on it as soon as I've got some free time (sometime in november)

**Comment by rafacouto** [comment]
> Docs are not quite ready for review, but here's a fix for the doc build failure.

Thank you for the tips :+1:

File: boards/arm/pinetime_devkit0/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+   :width: 600px
+   :align: center
+   :alt: Pine64 PineTime
+
+   Board Name (Credit: <owner>)
+
+
+References
+**********
+
```
Comment (dbkinder) [inline boards/arm/pinetime_devkit0/doc/index.rst]: Normally, links to other docs like this won't be displayed in the generated HTML. If you do want them to show up, add a directive before the first link:

```
.. target-notes::

.. _Pine64 wiki: https://wiki.pine64.org/index.php/PineTime
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by dbkinder — COMMENTED** [review]
OK.  Doc build errors are resolved.  Will there be more documentation coming?  I wandered around looking for some sample code for this platform and found some stuff written in Go on GitHub (https://github.com/aykevl/go-smartwatch/blob/master/smartwatch_pinetime-devkit0.go).  
What's the plan for this devkit and Zephyr?

**Comment by rafacouto** [comment]
> OK. Doc build errors are resolved. Will there be more documentation coming? I wandered around looking for some sample code for this platform and found some stuff written in Go on GitHub (https://github.com/aykevl/go-smartwatch/blob/master/smartwatch_pinetime-devkit0.go).

I will commit some basic sample in a few days (_samples/boards/pinetime_devkit0/blink_, right?).

> What's the plan for this devkit and Zephyr?

This is the board skeleton in order to developers are able to start their firmware apps with Zephyr.

**Comment by galak** [comment]
Any plans to update this PR?

**Comment by DWagenk** [comment]
@galak Thanks for cleaning up here. I guess closing this one and opening a new one once there's momentum again would be best. I'm still interested and did some testing based on this PR with the device, but can't forsee when I'll have time to pick it up again.

**Comment by sdorre** [comment]
May I help on this topic, I really would love to see the PineTime officially supported by Zephyr

**Comment by galak** [comment]
Closing this PR and lets focus on #26698

**Pull request closed without merging**